hvplot/plotting/scatter_matrix.py

```diff
@@ -4,10 +4,20 @@
 
 from ..converter import HoloViewsConverter
 from ..util import with_hv_extension
+from holoviews.operation.datashader import datashade as _datashade 
+from holoviews.operation.datashader import shade as _shade 
+from holoviews.operation.datashader import dynspread as _dynspread 
+from holoviews.operation.datashader import rasterize as _rasterize 
 
 
 @with_hv_extension
-def scatter_matrix(data, c=None, chart='scatter', diagonal='hist', alpha=0.5, **kwds):
+def scatter_matrix(data, c=None, chart='scatter', diagonal='hist', alpha=0.5, 
+    datashade=False, 
+    shade=False,
```

Comment:
I don't think there's a way that `shade=True` could work in a call to `scatter_matrix`; `shade()` expects an `Image` and converts it to an `RGB`, but here there are just points and histograms.  So I don't think this PR should include anything to do with `shade`, just `datashade` (the combination of rasterize and shade), `rasterize`, and `dynspread`. 

Maybe instead of `shade` you meant `spread`? Supporting `spread` would be useful, though you'd have to support at least some of its arguments the same way you support `max_px` for dynspread.

Reply:
Yes, you are right.  I meant to remove shade.  I had included that not knowing what it was before, and realized it didn't work.